packages/internal-playwright-helpers/index.ts

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { Page } from "@playwright/test";
+import { CognitoPage } from "./cognito";
+import { OpenIdPage } from "./open-id";
+import { getBrowserTestingEnvironment } from "@inrupt/internal-test-env";
+
+export { CognitoPage } from "./cognito";
+export { OpenIdPage } from "./open-id";
+export class TestPage {
+  page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async startLogin() {
+    const { idp } = getBrowserTestingEnvironment();
+    await this.page.fill("[data-testid=identityProviderInput]", idp);
```

Comment:
These helpers rely on these tag IDs, so it should probably be documented somewhere, or the associated components be exported (see comment https://github.com/inrupt/typescript-sdk-tools/pull/78#issuecomment-1324867490)

Reply:
@NSeydoux I just discussed this with @chelseapinka and we've a plan to tackle this:

- Add a `@inrupt/internal-playwright-testids` package with the testid strings
- Use those in both testApp's and `@inrupt/internal-playwright-helpers`